### ISSUE TITLE
Wrong color-matrix multiplying

### ISSUE THREAD

**Issue by hajimehoshi** [pixijs/pixi.js#3540]
In `_multiply`, 'translation' part of `a` is ignored.

```js
out[4] = (a[0] * b[4]) + (a[1] * b[9]) + (a[2] * b[14]) + (a[3] * b[19]);
```

should be

```js
out[4] = (a[0] * b[4]) + (a[1] * b[9]) + (a[2] * b[14]) + (a[3] * b[19]) + a[4];
```
.

Multiplying should be done as if `b` has fifth row like (0, 0, 0, 0, 1).

**Comment by GoodBoyDigital** [comment]
hi @hajimehoshi , thanks for sharing! Would you mind submitting a PR for this issue?

**Comment by hajimehoshi** [comment]
Not at all! I'll try.

**Comment by GoodBoyDigital** [comment]
👍

**Comment by hajimehoshi** [comment]
I've succeeded `npm install`but failed `npm test`

```
$ npm test

> pixi.js@4.3.2 test /Users/hajimehoshi/pixi.js
> floss --path test/index.js

[floss] caught outer exception: Cannot find module '../dist/pixi' Error: Cannot find module '../dist/pixi'
    at Module._resolveFilename (module.js:455:15)
    at Function.Module._resolveFilename (/Users/hajimehoshi/pixi.js/node_modules/electron-prebuilt/dist/Electron.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/hajimehoshi/pixi.js/test/index.js:4:1)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
npm ERR! Test failed.  See above for more details.
```

Am I missing something?

**Comment by GoodBoyDigital** [comment]
think you may need to run ```npm run build``` first

**Comment by englercj** [comment]
Looks like this was fixed since #3593 was merged.
